src/App.tsx

```diff
@@ -17,18 +38,38 @@ export const App: React.FC = () => {
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter
+                status={status}
+                handleChange={handleChange}
+                query={query}
+                setQuery={setQuery}
+              />
             </div>
 
             <div className="block">
-              <Loader />
-              <TodoList />
+              {!todos.length ? (
+                <Loader />
+              ) : (
+                <TodoList
+                  todos={visibleTodos}
+                  fnSelectTodo={(selectTodoId) => setSelectedTodoId(selectTodoId)}
+                  selectTodoId={selectedTodoId}
+                  fnSelectUser={(selectUserId) => setUserId(selectUserId)}
```

Comment:
Consider renaming the function 'fnSelectUser' to 'handleSelectUser' for better clarity.